deploy/journeyEngineStepFunction.asl.json

```diff
@@ -142,7 +142,6 @@
         "ipAddress.$": "$$.Execution.Input.ipAddress",
         "clientOAuthSessionId.$": "$$.Execution.Input.clientOAuthSessionId",
         "featureSet.$": "$$.Execution.Input.featureSet",
-        "lambdaInput.$": "$.lambdaInput",
```

Comment:
I think we still need this?

Reply:
yeah sure we needed that.....not sure what I was thinking.